client/src/pages/scheduling/Scheduler.js

```diff
@@ -42,17 +44,31 @@ const Scheduler = (props) => {
       .then(response => setEventType(response.data));
   }, []);
 
+  const todaysDate = new Date();
+  const tomorrowsDate = new Date(todaysDate);
+  tomorrowsDate.setDate(tomorrowsDate.getDate() + 1);
 
   const classes = useStyles();
-  const [selectedDay, setSelectedDay] = useState(null);
+  const [selectedDay, setSelectedDay] = useState({
+    day: tomorrowsDate.getDate(),
+    month: tomorrowsDate.getMonth() + 1,
```

Comment:
how come this needs to be + 1?

Reply:
if it's due to indices, maybes make a note of this in a comment

Reply:
yeah, regular JavaScript indexes the months starting with 0 whereas this library indexes it with 0, so December in the `.getMonth()` method returns 11, but to display it as December for this library, it should be 12 which is where the `...+ 1` adjusts it to.